static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -296,11 +301,47 @@ type SpecialFields = {
   'user.display': SpecialField;
 };
 
+const DownloadCount = styled('span')`
+  padding-left: 6px;
+`;
+
 /**
  * "Special fields" either do not map 1:1 to an single column in the event database,
  * or they require custom UI formatting that can't be handled by the datatype formatters.
  */
 const SPECIAL_FIELDS: SpecialFields = {
+  // This is a custom renderer for a field outside discover
+  // TODO - refactor code and remove from this file or add ability to query for attachments in
```

Comment:
Truncated comment? For attachments in ... something?

Reply:
Supposed to be `in Discover`, Updated!